src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/AssemblyAttributeProperties/AssemblyInfoProperties.cs

```diff
@@ -71,12 +71,12 @@ internal class AssemblyInfoProperties : DelegatedProjectPropertiesBase
         /// <summary>
         /// Get the value of a property.
         /// </summary>
-        public override async Task<string?> GetEvaluatedPropertyValueAsync(string propertyName)
+        public override async Task<string> GetEvaluatedPropertyValueAsync(string propertyName)
         {
             if (_attributeValueProviderMap.ContainsKey(propertyName) &&
                 !await IsAssemblyInfoPropertyGeneratedByBuild(propertyName))
             {
-                return await GetPropertyValueFromSourceAttributeAsync(propertyName);
+                return (await GetPropertyValueFromSourceAttributeAsync(propertyName)) ?? "";
```

Comment:
Behaviour change. Evaluated properties should not be null. Returns `""` instead.